src/page/blog/articles-page.jsx

```diff
@@ -22,13 +22,24 @@ export const Articles = () => {
   if (error) return <pre>{error.message}</pre>;
 
   return (
-    <div className="max-w-[1440px] mx-auto">
+    <div className="max-w-[1440px] mx-auto" data-test-id="articles-page">
       {isLoggedIn && (
-        <div className="flex mt-10 justify-end mr-32">
-          <a href="/draft" className="underline">
+        <div
+          className="flex mt-10 justify-end mr-32"
+          data-test-id="articles-page-navbar.loggedin"
+        >
+          <a
+            href="/draft"
+            className="underline"
+            data-test-id="articles-page-navbar.loggedin-drafts"
+          >
             Drafts
           </a>
-          <a href="/articles/create" className="underline ml-5">
+          <a
+            href="/articles/create"
+            className="underline ml-5"
+            data-test-id="articles-page-navbar.loggedin-new-entry"
```

Comment:
```suggestion
            data-test-id="articles-page-navbar-loggedin-new-entry"
```
If you feel like it's too long we don't have to use the whole id since this div is inside of another anyway. ie. the id could just be "navbar-loggedin-new-entry" or "loggedin-new-entry"